Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml

```diff
@@ -172,6 +172,22 @@
 		</Target>
 	</ECRelationshipClass>
 
+	<ECEntityClass typeName="Catchment" modifier="Sealed" displayLabel="Rainfall Set">
+		<BaseClass>anlyt:AnalyticalElement</BaseClass>
+		<ECProperty propertyName="Area" typeName="double" displayLabel="Area" category="HydraulicData" kindOfQuantity="rru:AREA"/>
+		<ECProperty propertyName="Tc" typeName="double" displayLabel="Time of Concentration" category="HydraulicData" kindOfQuantity="AECU:TIME"/>
+		<ECProperty propertyName="RationalC" typeName="double" displayLabel="Rational C" category="HydraulicDAta"/>
+	</ECEntityClass>
+
+	<ECRelationshipClass typeName="CatchmentOwnsOutflowNode" strength="referencing">
```

Comment:
BIS is strict separating "Owns" semantics from other kinds of relationships.

"Owns" means parent-child (i.e. cascading-delete). "Owns" relationships must subclass bis:ElementOwnsChildElements (which is a foreign-key/navigation property kind of relationship).

If the desired relationship is just a simple foreign-key (i.e. many-to-one) without "owns" semantics, you could introduce it as a navigation property of the Catchment. e.g.

<ECNavigationProperty propertyName="OutflowNode" relationshipName="CatchmentDischargesToOutflowNode" direction="forward" displayLabel="Outflow Node" description="..." />

    <ECRelationshipClass typeName="CatchmentDischargesToOutflowNode" displayLabel="Catchment discharges to Outflow Node" strength="referencing" strengthDirection="forward" description="...">
        <!-- @see Catchment.Outfall ECNavigationProperty -->
        <Source multiplicity="(0..1)" roleLabel="discharges" polymorphic="false">
            <Class class="Catchment"/>
        </Source>
        <Target multiplicity="(0..*)" roleLabel="is discharged by" polymorphic="true">
            <Class class="net:INode" />
        </Target>
    </ECRelationshipClass>

Reply:
the catchment doesn't own the outflow node - it references it .... if the catchment is deleted the outflow node should not be deleted....

the catchment can only reference one outflow node...